CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+**Table of Contents**
+<!-- TOC depthFrom:2 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Versioning](#versioning)
+- [Branches](#branches)
+- [Pull Requests](#pull-requests)
+	- [Working on a Pull Request](#working-on-a-pull-request)
+	- [Reviewing a Pull Request](#reviewing-a-pull-request)
+	- [Merging a Pull Request](#merging-a-pull-request)
+- [Zenhub Issue Board](#zenhub-issue-board)
+	- [Pipelines](#pipelines)
+- [Reporting A Bug](#reporting-a-bug)
+	- [Before Submitting A Bug Report](#before-submitting-a-bug-report)
+	- [How Do I Submit A Bug Report?](#how-do-i-submit-a-bug-report)
+
+<!-- /TOC -->
+## Versioning
+
+This project uses [Semantic Versioning](http://semver.org/) or _SemVer_ for short. If you do not know what SemVer is, a detailed explanation is available on their website.
+Any new releases must adhere to this versioning scheme.
+
+## Branches
+Our project currently consists of a _master_ and several _feature_ branches. 
+The master branch always contains a stable version of the project. Only commits with small, insignificant changes should be done directly on the master branch. 
+
+Any other changes should first be implemented and tested on a feature branch. The name of this feature branch should closely describe the added changes.
+
+If you want to add your changes to the main branch you can do so by proposing a [pull request](#pull-requests).
+
+## Pull Requests
+Any major changes to existing or the addition of new features or artifacts should be done through pull requests or PRs.
+
+### Working on a Pull Request
+PRs with unfinished features should have a [WIP] tag at the beginning of their title. This shows that this feature is still in progress and not ready to be merged.
+
+### Reviewing a Pull Request
+When a feature is considered finished it should be **reviewed** before merging the corresponding PR. During this stage the [WIP] label in the title should be removed.
```

Comment:
Maybe use labels instead or in addition.

Reply:
The [WIP] tag is a common practice. I do not think we need labels for this.